static/scripts/ckeditor/plugins/ckeditor5-filebrowser/src/filebrowser.js

```diff
@@ -87,8 +89,10 @@ export default class FileBrowserPlugin extends Plugin {
 							src: imageUrl,
 							alt: imageAltText,
 						});
-
-						editor.model.insertContent(imageElement, editor.model.document.selection);
+						const lastOpenedEditorId = document.getElementById('editor-id').value;
```

Comment:
It seems that there must be a nicer way to find out the current editor, but I couldn't find one now either. If it works I would go with it for now.

Reply:
Me either...